lib/parse/base.js

```diff
@@ -159,8 +159,6 @@ Base.prototype.getFileSource = function (filename) {
 			header: false,
 			bare: true
 		});
-	} else if (filename.match(this.jsxExtRegEx)) {
```

Comment:
> Incidentally, are you a maintainer on madge, or - what's your interest in it?

Just a fan. I also like seeing if there are any problems with the detective usage.

> react-tools removed

I don't have enough context to know whether or not it's safe to remove this. cc @pahen